### PR TITLE
CRM-3336

### PR DESCRIPTION
**Pull request by benwells**
- separating out label and text so that label can be styled separately by labelClass
 - the old "label" prop will become the "text" prop.   I will need to update all implementations of <Marker /> throughout app-react.

### CODE REVIEW

File: src/Components/Marker.jsx [hunk 1]
```diff
@@ -12,7 +12,9 @@ module.exports = React.createClass({
 
   propTypes: {
     type:       React.PropTypes.string,
-    label:      React.PropTypes.string.isRequired,
+    label:      React.PropTypes.string,
+    text:      React.PropTypes.string.isRequired,
```
Comment (Pinesy) [inline src/Components/Marker.jsx]: Dun dun dun, need a space here :)

File: src/Components/Marker.jsx [hunk 2]
```diff
@@ -39,9 +42,17 @@ module.exports = React.createClass({
...
+      label = (
+        <span className={this.props.labelClass}>{this.props.label} </span>
+      );
+    }
+
     return (
       <span className={markerClass}>
-        <span>{this.props.label} </span>
+        {label}
+        <span>{this.props.text} </span>
```
Comment (Pinesy) [inline src/Components/Marker.jsx]: is there meant to be an extra space after `{this.props.text}`?

Comment (benwells) [inline src/Components/Marker.jsx]: Yes

File: src/Components/Marker.jsx (hunk 1 above)
Comment (benwells) [inline src/Components/Marker.jsx]: arg!

File: src/Components/Marker.jsx (hunk 2 above)
Comment (Pinesy) [inline src/Components/Marker.jsx]: 👍

**Comment by Pinesy** [comment]
LGTM

**Pull request closed without merging**